### DISCUSSION TITLE
Database error saving new user 500

### DISCUSSION THREAD

**Discussion by rajatdhoot123** [supabase/supabase#13834] [Questions]
code: 500
error_id: "7ba6562903709aa7-NAG"
msg: "Database error saving new user"

**Comment by GaryAustin1** [comment]
I moved this from issues as a bug as you have not marked you have done research on it yet, and it appears to be a typical coding issue at this point.  Please feel free to reopen as a bug after seeing below link and asking for help.
Please see https://github.com/orgs/supabase/discussions/13043 before declaring it a bug.
